packages/selected-network-controller/package.json

```diff
@@ -34,6 +34,7 @@
     "@metamask/base-controller": "^4.1.1",
     "@metamask/json-rpc-engine": "^7.3.2",
     "@metamask/network-controller": "^17.2.0",
+    "@metamask/permission-controller": "^8.0.0",
```

Comment:
This seems like a pretty significant cost, just to get the type `(string) => boolean`. I don't think inlining that type would incur any significant maintenance burden.

Reply:
done here: https://github.com/MetaMask/core/pull/3908/commits/ecb5e72a77608459747c279a2e14aad4c94b63a1